src/pts/pyspark/orphanet.py

```diff
@@ -30,24 +30,27 @@
 
 
 def orphanet(
-    source: str,
+    source: dict[str, str],
     destination: str,
     properties: dict[str, str],
 ) -> DataFrame:
     spark = Session(app_name='orphanet', properties=properties)
-    efo_version = properties['efo_version']
-    cores = int(properties.get('ontology_cores', 1))
 
-    logger.info(f'parse XML from {source} into a list of dictionaries')
-    orphanet_disorders = parse_orphanet_xml(source)
+    logger.info(f'parse XML from {source['evidence']} into a list of dictionaries')
```

Comment:
I think this would crash because of the quotes
```suggestion
    logger.info(f'parse XML from {source["evidence"]} into a list of dictionaries')
```